internal/pkg/cli/flag.go

```diff
@@ -264,6 +266,8 @@ Defaults to all logs. Only one of end-time / follow may be used.`
 	deleteSecretFlagDescription      = "Deletes AWS Secrets Manager secret associated with a pipeline source repository."
 	svcPortFlagDescription           = "The port on which your service listens."
 
+	ingressTypeFlagDescription = "Required for Request-Driven Web Service. Who can send traffic to your service."
```

Comment:
This wording isn't quite right, either, but may be slightly clearer?
```suggestion
	ingressTypeFlagDescription = "Required for a Request-Driven Web Service. Allowed source of traffic to your service."
```